### ISSUE TITLE
Serious vulnerability in Axios

### ISSUE THREAD

**Issue by j-rewerts** [PRB0t/PRB0t#50]
There was [an issue ](https://github.com/axios/axios/issues/2183)reported with Axios yesterday. It looks like one of our libraries depends on it, so we should update.

**Comment by j-rewerts** [comment]
@M3kH Feel free to assign this to me.

**Comment by j-rewerts** [comment]
`github-api` uses Axios, so I'll resolve the vuln [upstream](https://github.com/github-tools/github/issues/565).

**Comment by j-rewerts** [comment]
I've submitted the PR [here](https://github.com/github-tools/github/pull/567).

**Comment by M3kH** [comment]
Yep unfortunately that is the warning on the dependency, but not that much we can do.
Unless `github-api` updates it.

Something that is worth considering is maybe not having dependency to `github-api`.
And maybe create implementations for multiple hosted git vendors.

But this would require major re-work.
